src/main/java/ita/softserve/course_evaluation/service/impl/UserServiceImpl.java

```diff
@@ -15,8 +18,37 @@
 	private UserRepository userRepository;
 
 	@Override
-	public List<User>  readAll() {
-		return userRepository.findAll();
+	public List<UserDto>  readAll() {
+		return UserDtoMapper.toDto(userRepository.findAll());
+	}
+
+	@Override
+	public  UserDto readById(long id ) { return UserDtoMapper.toDto(userRepository.findUserById( id).get());}
+
+	@Override
+	public UserDto   readByFirstName(String firstName) {return UserDtoMapper.toDto(userRepository.findUserByFirstName(firstName).get());}
+
+	@Override
+	public void createUser(UserDto dto) {
+		userRepository.save(UserDtoMapper.fromDto(dto));
```

Comment:
The create method usually returns created entity

Reply:
corrected